diff --git a/fs/read_write.c b/fs/read_write.c
--- a/fs/read_write.c
+++ b/fs/read_write.c
@@ -718,8 +718,10 @@ ssize_t ksys_read(unsigned int fd, char __user *buf, size_t count)
 	return ret;
 }

-#ifdef CONFIG_KSU_SUSFS
+#ifdef CONFIG_KSU_MANUAL_HOOK
 extern bool ksu_init_rc_hook __read_mostly;
 extern __attribute__((cold)) void ksu_handle_sys_read(unsigned int fd);
 #endif

 SYSCALL_DEFINE3(read, unsigned int, fd, char __user *, buf, size_t, count)
 {
-#ifdef CONFIG_KSU_SUSFS
+#ifdef CONFIG_KSU_MANUAL_HOOK
 	if (unlikely(ksu_init_rc_hook))
 		ksu_handle_sys_read(fd);
 #endif

 	return ksys_read(fd, buf, count);
 }
